README.md

```diff
@@ -1,3 +1,5 @@
+Repo used for adding Swedish translation and usage.
+
```

Comment:
This should not be added to the PR, we can merge this to the main repo.

Reply:
Ah so that's why forks usually doesn't have descriptions in the README. I guess I could've excluded it from the files, but I just deleted it from the PR instead.